src/cheri/insns/csrr_32bit.adoc

```diff
@@ -46,44 +46,24 @@ Encoding::
 include::wavedrom/csr-instr.adoc[]
 
 Description::
-These are standard RISC-V CSR instructions with extended functionality for
-accessing CLEN-wide CSRs (see xref:all_capability_CSRs[xrefstyle=short]),
-including extended CSRs with actions as shown in
+These are standard RISC-V CSR instructions with extended functionality for accessing CLEN-wide CSRs.
 +
 For CLEN-wide CSRs, the full capability is read into `cd` in {cheri_cap_mode_name}.
 In {cheri_int_mode_name}, the address field is instead read into `rd`.
 +
-Unlike <<CSRRW_CHERI>>, these instructions only update the address field and the valid tag
-as defined in xref:extended_CSR_writing[xrefstyle=short] when writing
-CLEN-wide CSRs regardless of the execution mode. The final address to
-write to the capability CSR is determined as defined by RISC-V for these
-instructions.
+Unlike <<CSRRW_CHERI>>, these instructions only update the address field and the valid tag as defined in xref:extended_CSR_writing[xrefstyle=short] when writing CLEN-wide CSRs regardless of the execution mode.
+The CSR update is made using <<SCADDR>> semantics.
 +
-See xref:aliased_CSRs[xrefstyle=short] for a list of CLEN-wide CSRs and
-xref:extended_CSR_writing[xrefstyle=short] for the action taken on writing an XLEN-wide value to each one.
-+
-If `cd` is `c0` (or `rd` is `x0`), then <<CSRRWI_CHERI,CSRRWI>> shall not read the CSR and
-and shall not cause any of the side effects that might occur on a CSR read. If
-`rs1` is `x0` for <<CSRRS_CHERI,CSRRS>> and <<CSRRC_CHERI,CSRRC>>, or `imm` is 0 for <<CSRRSI_CHERI,CSRRSI>> and
-<<CSRRCI_CHERI,CSRRCI>>, then the instruction will not write to the CSR at all, and so shall
-not cause any of the side effects that might otherwise occur on a CSR write.
```

Comment:
I know this is already defined in the base ext but I think it might be useful to either have a xref to the relevant section or keep this.

Reply:
I've address this referring to Zicsr rules